thirdparty/kyaml/runfn/runfn.go

```diff
@@ -248,50 +219,27 @@ func (r RunFns) runFunctions(
 	return nil
 }
 
-// getFunctionsFromInput scans the input for functions and runs them
-func (r RunFns) getFunctionsFromInput(nodes []*yaml.RNode) ([]kio.Filter, error) {
-	if *r.NoFunctionsFromInput {
+// getFunctionsFromFunctionPath returns the function read from r.FunctionPath
+func (r RunFns) getFunctionsFromFunctionPaths() ([]kio.Filter, error) {
+	if r.FunctionPath == "" {
 		return nil, nil
 	}
-
-	buff := &kio.PackageBuffer{}
-	err := kio.Pipeline{
-		Inputs:  []kio.Reader{&kio.PackageBuffer{Nodes: nodes}},
-		Filters: []kio.Filter{&runtimeutil.IsReconcilerFilter{}},
-		Outputs: []kio.Writer{buff},
-	}.Execute()
+	f, err := os.Open(r.FunctionPath)
```

Comment:
I think there is a misunderstanding what --fn-config means here. let's have an offline chat about it.

Reply:
Ok I see. I think now I understand it. The content in the file should be in `functionConfig` field in the function input right?

Reply:
Yes :)

Reply:
Updated